sites.json

```diff
@@ -4753,6 +4753,16 @@
         ]
     },
 
+    {
+		"name": "Pocoyo Club",
+		"url": "https://www.pocoyo.com/en/club/profile",
+		"difficulty": "easy",
+		"notes": "You can also just unsubscribe from \"newsletters and other commercial, promotional and advertising communications from Zinkia Entertainment S.A. and/or from third parties\" at the Notifications tab",
```

Comment:
These notes do not describe how to delete an account; the link is not to a direct deletion page. They also do not explain how unsubscribing from newsletters et. al. gives the same behavior.